composer.lock

```diff
@@ -4,20 +4,20 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "68ec110aab4b7d444c66d090d7252e4a",
+    "content-hash": "8c1127e8c3f580a05e127400b65efd46",
```

Comment:
How did you generate this in the end? We should document/automate this so that it's easy for someone else to do this.

Reply:
I followed along this link: https://devcenter.heroku.com/articles/deploying-php
Ended up going with composer update. I'm not sure how to automate that just yet. It appears as though the users get this lock file when they clone the project. If that's the case why would we need to automate this? Obviously, I'm not sure what "content-hash" is up to, so I'm happy to dig through this some more.

Reply:
The part that I want to automate is the composer.lock generation so that you, me, Jonny, etc can generate it consistently.

Right now, having to do things locally is not consistent. We need a consistent way to do this just like how we use docker to standardize our local development.

I would like us to use https://hub.docker.com/r/library/composer/. I've done it and I can show you how to do this.